src/NuGet/Microsoft.Orleans.Core.nuspec

```diff
@@ -21,7 +21,7 @@
       <dependency id="Newtonsoft.Json" version="9.0.1" />
       <dependency id="System.Collections.Immutable" version="1.3.1" />
       <dependency id="System.Reflection.Metadata" version="1.4.2" />
-      <dependency id="Microsoft.Extensions.DependencyInjection" version="1.1.0" />
+      <dependency id="Microsoft.Extensions.DependencyInjection" version="1.1.1" />
```

Comment:
Well, the point is that there are no breaking changes, but the reason to do it here, is that in 1.1.1 they solved a race condition bug that ONLY happens if you are using scopes. So there's no value of updating *unless* we use scopes, and this PR without the update should not be merged *unless* we upgrade to 1.1.1 and avoid the race.